Services/UI/OJS.Services.Ui.Business/Validation/ValidationMessages.cs

```diff
@@ -23,5 +23,35 @@ public static class Submission
         public const string NotFound = "Submission not found!";
 
         public const string NotMadeByUser = "You can only view your own submissions!";
+
+        public const string SubmissionTypeNotFound = "Wrong submission type!";
+
+        public const string BinaryFilesNotAllowed = "This submission type does not allow sending binary files";
```

Comment:
Maybe "." at the end of all strings of this file, in order to keep the convention.

Reply:
Done.